src/main/java/uk/gov/justice/hmpps/prison/repository/ScheduleRepository.kt

```diff
@@ -206,6 +206,20 @@ class ScheduleRepository : RepositoryBase() {
     return schedules.toPrisonerSchedules()
   }
 
+  fun getScheduledTransfersForPrisoner(
+    prisonerNumber: String?,
+  ): List<PrisonerSchedule> {
```

Comment:
very minor but you may as well drop the optional ? given it can't get here an be null